src/modules/commentcounter.test.js

```diff
@@ -0,0 +1,16 @@
+import commentCounterFunction from './commentCounter.js';
```

Comment:
## Required
kindly fix linters errors
There is no commentCounter file.<img src="https://media0.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>